Sources/TuistGenerator/Generator/ProjectDescriptorGenerator.swift

```diff
@@ -295,8 +295,14 @@ final class ProjectDescriptorGenerator: ProjectDescriptorGenerating {
     private func generateAttributes(project: Project) -> [String: Any] {
         var attributes: [String: Any] = [:]
 
-        /// ODR tags
-        let tags = project.targets.map { $0.resources.resources.map(\.tags).flatMap { $0 } }.flatMap { $0 }
+        // On Demand Resources tags
+        var tags = project.targets.map { $0.resources.resources.map(\.tags).flatMap { $0 } }.flatMap { $0 }
```

Comment:
TODO: Project level list of all known tags (`KnownAssetTags` attribute) comes from several sources:

1. All targets' Initial Install tags
2. All targets' Prefetch Order tags
3. File resources tags - default On Demand tags
4. Resources in asset catalogs:
4.1 Data file (`.dataset`)
4.2 Image (`.imageset`)
4.3 SpriteKit texture atlas (`.textureset`)
4.4 Apple TV Image Stack (`.imagestack`)
Gathering known tags from sources 1-3 is done but we have to parse `Contents.json` files in asset catalogs in their directories described above.
Here is an example of an image asset `Contents.json`:
```
{
  "images" : [
    {
      "filename" : "img.png",
      "idiom" : "universal",
      "scale" : "1x"
    },
    {
      "idiom" : "universal",
      "scale" : "2x"
    },
    {
      "idiom" : "universal",
      "scale" : "3x"
    }
  ],
  "info" : {
    "author" : "xcode",
    "version" : 1
  },
  "properties" : {
    "on-demand-resource-tags" : [
      "my-tag"
    ]
  }
}
```